build/_build/Build.Steps.cs

```diff
@@ -925,6 +1003,7 @@ projectPath switch
                         "Samples.AspNetCoreMvc21" => Framework == TargetFramework.NETCOREAPP2_1,
                         "Samples.AspNetCoreMvc30" => Framework == TargetFramework.NETCOREAPP3_0,
                         "Samples.AspNetCoreMvc31" => Framework == TargetFramework.NETCOREAPP3_1,
+                        "Samples.AspNetCore5" => Framework == TargetFramework.NET5_0 || Framework == TargetFramework.NETCOREAPP3_1 || Framework == TargetFramework.NETCOREAPP3_0,
```

Comment:
I believe what we had would work, but has not been tested yet. The next phase of development will focus on QA, including verifying .NET 2.1 / 2.0 support.

I have logged https://datadoghq.atlassian.net/browse/APPSEC-987 for this.

Reply:
My biggest concern is that it doesn't break anything on .NET Core 2.1 (including netfx on aspnetcore 2.1). We're doing extra instrumentation regardless of if app sec is enabled, so would be good to know that nothing's in the tracer is broken at least 🙂

Reply:
I would have thought the existing integration tests would cover non-regression for the tracer when app sec is disabled.

But extra tests are never a bad thing, so I added a separate 2.1 test project and it made be discover that blocking doesn't work under 2.1 because of internal changes to asp.net core. Fortunately quite easy to fix see c19cb62c1552342c1f1839a48d25626538db42be